packages/grafana-ui/src/saga-themes/code/themes/dark.ts

```diff
@@ -0,0 +1,67 @@
+export default {
+  color: {
+    background: {
+      ui: {
+        canvas: 'hsl(220 11.8% 7.2%)',
+        primary: 'hsl(220 11.8% 10.8%)',
+        secondary: 'hsl(220 11.8% 15.6%)',
+        tertiary: 'hsl(220 11.8% 28.8%)',
+        overlay: 'hsl(220 11.8% 10.8% / 0.4)',
+      },
+      system: {
+        success: 'hsl(149 66% 5.4%)',
+        warning: 'hsl(36 100% 9%)',
+        error: 'hsl(336 87.4% 7.87%)',
+        info: 'hsl(220 67.2% 9.81%)',
+      },
+    },
+    border: {
+      ui: {
+        weak: 'hsl(220 11.8% 15.6%)',
+        medium: 'hsl(220 11.8% 28.8%)',
+        strong: 'hsl(220 11.8% 45.6%)',
+        divider: 'hsl(220 11.8% 28.8%)',
+      },
+      system: {
+        success: 'hsl(149 66% 22.8%)',
+        warning: 'hsl(36 100% 38%)',
+        error: 'hsl(336 87.4% 33.2%)',
+        info: 'hsl(220 67.2% 41.4%)',
+      },
+    },
+    content: {
+      primary: 'hsl(220 11.8% 98.4%)',
+      secondary: 'hsl(220 11.8% 76.8%)',
+      disabled: '#8D95A5',
+      link: 'hsl(220 67.2% 63.6%)',
+      system: {
+        success: {
+          heading: 'hsl(149 66% 74.8%)',
+          body: 'hsl(149 66% 91.6%)',
+        },
+        warning: {
+          heading: 'hsl(36 100% 82%)',
+          body: 'hsl(36 100% 94%)',
+        },
+        error: {
+          heading: 'hsl(336 87.4% 79.7%)',
+          body: 'hsl(336 87.4% 93.2%)',
+        },
+        info: {
+          heading: 'hsl(220 67.2% 83.6%)',
+          body: 'hsl(220 67.2% 94.5%)',
+        },
```

Comment:
I find this naming a lot more confusing than the current theme (where we have theme.colors.text for text colors), 

here would not have a clue what counts as "content" or how to use these